packages/cli-e2e/entrypoints/dockerX11Entry.sh

```diff
@@ -13,15 +14,17 @@ npm run build
 export UI_TEMPLATE_VERSION=0.0.0
 npm set registry http://verdaccio:4873
 yarn config set registry http://verdaccio:4873
+ts-node --transpile-only ./packages/cli-e2e/utils/npmLogin.ts
 
 npm run npm:bump:template -- -- $UI_TEMPLATE_VERSION
 
-npm run npm:publish:template
+npm run npm:publish:template:local
```

Comment:
Needed a new a script 'cause the verdaccio host changes due to docker-compose shenanigans